### PR TITLE
Bin Feature flags

### PR DESCRIPTION
**Pull request by darokel**
It's about time we sort this out. This PR removes all unused feature flags from the code base. All the features are live in production so the flags are pointless. 

Corresponding PR to remove them from environments in Puppet is here https://github.com/moneyadviceservice/scripts/pull/463

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -22,10 +22,6 @@ def not_implemented
 
     get '/sitemap', to: 'sitemap#index'
 
-    if Feature.active?(:improvements)
-      mount Feedback::Engine => '/improvements'
```
Comment (tomas-stefano) [inline config/routes.rb]: Is this good to be deleted?

Comment (tomas-stefano) [inline config/routes.rb]: Ok I found more in the bottom.

Comment (darokel) [inline config/routes.rb]: It hasn't been deleted, just moved down to the bottom

**Comment by tomas-stefano** [comment]
:+1:

**Comment by benbarnett** [comment]
Testy passy :+1:

File: features/support/env.rb [hunk 2]
```diff
@@ -64,21 +64,19 @@
...
+#     Devise.class_variable_set(:@@warden_configured, false)
+#     Devise.configure_warden!
+#
+#     block.call
+#
+#     Devise.regenerate_helpers!
+#     Devise.class_variable_set(:@@warden_configured, false)
+#     Devise.configure_warden!
+#   end
+# end
```
Comment (asmega) [inline features/support/env.rb]: can you just delete this code instead of commenting it out

Comment (darokel) [inline features/support/env.rb]: Done

**Comment by asmega** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
:+1:

**Pull request merged**